_episodes/09-remotes.md

```diff
@@ -200,9 +66,17 @@ Pulling has no effect in this case because the two repositories are already
 synchronized.  If someone else had pushed some changes to the repository on
 GitHub, though, this command would download them to our local repository.
 
+> ## GitHub References
+>
+> Below are a list of references you may want to review the next time you create a new repository on Github.
+>
+> * https://help.github.com/articles/create-a-repo/
```

Comment:
I would put these up by the 1-4 so someone reading this won't be too lost.

Reply:
What about above the steps? Then we don't break the 'flow', but i totally agree they're pushed down fairly far if folks want the quick reference.

Reply:
Yeah, that works too.